_data/Tasmin153/Tasmin153.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Mahbuba Tasmin
+institution: North South University
+profile_pic: passport_Mahbuba Tasmin.png
```

Comment:
Hey @Tasmin153 , just mention your picture file name with the exact extension. So here it goes as:
`profile_pic: Mahbuba_Tasmin_Avatar.jpg`